sktime/forecasting/ets.py

```diff
@@ -444,7 +450,7 @@ def _predict_interval(self, fh, X=None, coverage=None):
 
         start, end = valid_indices[[0, -1]]
         prediction_results = self._fitted_forecaster.get_prediction(
-            start=start, end=end
+            start=start, end=end, random_state=self.random_state
```

Comment:
is this only used here this `get_prediction` method?

Reply:
Yes , actually in PR #2315 ets needs a random_state to be set so as to avoid fluctuations in internal predictions random_state can be set only in the simulate method get_prediction can pass parameters to the simulate method through (**simulate kwargs)
![image](https://user-images.githubusercontent.com/81592570/163594641-65785a83-0297-4ce5-9464-6bf0b8ca1204.png)

Reply:
are we passing that then also to `simulate`?

Reply:
Not separately ; no

Reply:
ah `get_prediction` is a `statsmodels` method, so then I think its fine